clients/src/test/java/org/apache/kafka/clients/NodeApiVersionsTest.java

```diff
@@ -54,6 +54,18 @@ public void testUnknownApiVersionsToString() {
         assertTrue(versions.toString().endsWith("UNKNOWN(337): 0 to 1)"));
     }
 
+    @Test(expected = UnsupportedVersionException.class)
+    public void testEnsureUsableUnusuableVersion() {
```

Comment:
LOL, looks like my mind got stuck somewhere between typing "unusual" and "unusable".